### PR TITLE
LWIMIQ: Yet Another move by Euwe!

### PR DESCRIPTION
**Pull request by lpichler**

### CODE REVIEW

**Comment by JPrause** [comment]
@syncrou @jrafanie @imtayadeway can you each review the latest blog entry.
Thanks @lpichler for your blog entry.

**Comment by JPrause** [comment]
LGTM,....I like the ""So a Hungarian, a Czech and an American are creating a pull request" ... no, it isn't a begining of a joke". Ha!

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- [47 PRs][manageiq-ui-classic PRs merged] merged in [ManageIQ UI classic][manageiq-ui-classic] controller repo.
+- [17 PRs][manageiq-ui-service PRs merged] merged in [ManageIQ UI Service][manageiq-ui-service] javascript repo.
+- [17 PRs][manageiq-providers-amazon PRs merged] merged in [ManageIQ Providers Amazon][manageiq-providers-amazon] repo.
+- [7 PR][manageiq-gems-pending PRs merged] merged in [ManageIQ Gems Pending][manageiq-gems-pending] repo.
+- [5 PR][manageiq-providers-vmware PRs merged] merged in [ManageIQ Providers VMWare][manageiq-providers-vmware] repo.
+- [3 PR][manageiq-providers-lenovo PRs merged] merged in [ManageIQ Providers Lenovo][manageiq-providers-lenovo] repo.
+- [1 PR][manageiq-providers-azure PRs merged] merged in [ManageIQ Providers Azure][manageiq-providers-azure] repo.
+
+Last line of ruby code in last week have number `253 718` in [ManageIQ][manageiq] core repo and `3 810` in [ManageIQ UI classic][manageiq-ui-classic] controller repo.
+Javascript coders this week added `432 642nd` line.
```
Comment (syncrou) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: @lpichler - Really ❤️  the ruby code numbers!

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+## Deleted
+
+Best code is no code. Our contributors are coding hand by hand with this statement, see their deletions: 
+
+- [James](https://github.com/jameswnl) - [Remove uncalled miq_server lets](https://github.com/ManageIQ/manageiq/pull/14383): `+1,181 −16,115`
+- [Tim](https://github.com/imtayadeway) - [Remove uncalled miq_server lets](https://github.com/ManageIQ/manageiq/pull/14355): `+0 −12`
+- [Robin](https://github.com/PanSpagetka) - [Remove unnecessary chart code](https://github.com/ManageIQ/manageiq/pull/13470): `+7 −61`
+- [Nick](https://github.com/NickLaMuro) - [Remove `hide_action` patch to StaticController](https://github.com/ManageIQ/manageiq-ui-classic/pull/660): `+0 −5`
+
+Should be noted there we like pets and I assure you that [Martin](https://github.com/himdel) has the same view of the matter even when he published PR called [Remove dead @cats](https://github.com/ManageIQ/manageiq-ui-classic/pull/664) (No one was harmed during coding/review of this PR)
```
Comment (syncrou) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: s/there/that ?

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+My name is [Libor](https://github.com/lpichler "Can you guess what word is highlighted in my avatar?") and I am part of Platform Team in Brno.
+
+It's that time again. Time to recap of last week in ManageIQ!
+
+## Featured
+This week we were busy with moving last features to Euwe for upcoming 5.7.2 release!
+
+Let's start typically - with some numbers:
+- [88 PRs][manageiq PRs merged] merged in [ManageIQ][manageiq] core repo.
```
Comment (syncrou) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: Do you want to point to the [ManageIQ](https://github.com/manageiq/manageiq) repository page or the ManageIQ organization page?

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- [47 PRs][manageiq-ui-classic PRs merged] merged in [ManageIQ UI classic][manageiq-ui-classic] controller repo.
+- [17 PRs][manageiq-ui-service PRs merged] merged in [ManageIQ UI Service][manageiq-ui-service] javascript repo.
+- [17 PRs][manageiq-providers-amazon PRs merged] merged in [ManageIQ Providers Amazon][manageiq-providers-amazon] repo.
+- [7 PR][manageiq-gems-pending PRs merged] merged in [ManageIQ Gems Pending][manageiq-gems-pending] repo.
+- [5 PR][manageiq-providers-vmware PRs merged] merged in [ManageIQ Providers VMWare][manageiq-providers-vmware] repo.
+- [3 PR][manageiq-providers-lenovo PRs merged] merged in [ManageIQ Providers Lenovo][manageiq-providers-lenovo] repo.
+- [1 PR][manageiq-providers-azure PRs merged] merged in [ManageIQ Providers Azure][manageiq-providers-azure] repo.
+
+Last line of ruby code in last week have number `253 718` in [ManageIQ][manageiq] core repo and `3 810` in [ManageIQ UI classic][manageiq-ui-classic] controller repo.
```
Comment (syncrou) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: Same question in regards to the ManageIQ repo vs Organization page.

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+Let's start typically - with some numbers:
+- [88 PRs][manageiq PRs merged] merged in [ManageIQ][manageiq] core repo.
+- [47 PRs][manageiq-ui-classic PRs merged] merged in [ManageIQ UI classic][manageiq-ui-classic] controller repo.
+- [17 PRs][manageiq-ui-service PRs merged] merged in [ManageIQ UI Service][manageiq-ui-service] javascript repo.
+- [17 PRs][manageiq-providers-amazon PRs merged] merged in [ManageIQ Providers Amazon][manageiq-providers-amazon] repo.
+- [7 PR][manageiq-gems-pending PRs merged] merged in [ManageIQ Gems Pending][manageiq-gems-pending] repo.
+- [5 PR][manageiq-providers-vmware PRs merged] merged in [ManageIQ Providers VMWare][manageiq-providers-vmware] repo.
+- [3 PR][manageiq-providers-lenovo PRs merged] merged in [ManageIQ Providers Lenovo][manageiq-providers-lenovo] repo.
+- [1 PR][manageiq-providers-azure PRs merged] merged in [ManageIQ Providers Azure][manageiq-providers-azure] repo.
+
```
Comment (syncrou) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: Possibly s/PR/PRs/ for line 23,24,25

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+## Other Moves
+[Marcel](https://github.com/durandom) moved [azure settings](https://github.com/ManageIQ/manageiq/pull/14345) to [azure provider](https://github.com/ManageIQ/manageiq-providers-azure/pull/40)
+
+[Adam](https://github.com/agrare) moved [fog-vcloud-director gem](https://github.com/ManageIQ/manageiq/pull/14358) to [manageiq-providers-vmware gemspec](https://github.com/ManageIQ/manageiq-providers-vmware/pull/28)
+
+[James](https://github.com/jameswnl) moved [Credential's API_ATTRIBUTES to the shared one](https://github.com/ManageIQ/manageiq/pull/14334)
+
+## Docs
+[Ladas](https://github.com/lsmola) is showing to us that developer can also create nice documentation.
```
Comment (syncrou) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: Ladas page was returning a 404

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+## Fixed
+Time is most important physical quantity on the world and also [Keenan](https://github.com/kbrock) is aware of this fact. It is visible in his [peformance PR](https://github.com/ManageIQ/manageiq/pull/14285 "second part") to speed up for generating widgets.
+
+[Beni](https://github.com/cben) with his [Container TLS fixes](https://github.com/ManageIQ/manageiq-ui-classic/pull/670) is fixing ability to create and edit container's provider without Hawkular.
+
+Broken chargeback has been fixed by [Allen](https://github.com/ManageIQ/manageiq-ui-service/pull/591) in [ManageIQ UI Service][manageiq-ui-service] repo.
+
+## New
+[Hungarian](https://github.com/skateman), [Czech](https://github.com/martinpovolny) and [Ame](https://github.com/abellotti)[ric](https://github.com/Fryguy)[ans](https://github.com/yrudman) are creating a pull request ... it is not begining of joke, they worked together on one PR, concretely they added [option for disabling asynchronous](https://github.com/ManageIQ/manageiq/pull/13898) notifications.
```
Comment (syncrou) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: LOL!

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+*Warning!!! The PR is not for people with hexakosioihexekontahexaphobia*
+
+[Erez](https://github.com/ManageIQ/manageiq/pull/13704) added [External Logging Link](https://github.com/ManageIQ/manageiq/pull/13704) for container nodes.
+
+
+## Deleted
+
+Best code is no code. Our contributors are coding hand by hand with this statement, see their deletions: 
+
+- [James](https://github.com/jameswnl) - [Remove uncalled miq_server lets](https://github.com/ManageIQ/manageiq/pull/14383): `+1,181 −16,115`
```
Comment (imtayadeway) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: James' PR link seems to have the same description as the one below but I don't think that's right?

**Comment by imtayadeway** [comment]
LGTM if you can just check the description of James' PR :+1:

**Comment by JPrause** [comment]
@lpichler let me know when you have all edits completed, then we can merge.

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md (hunk 3 above)
Comment (lpichler) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: yes, repo

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md (hunk 5 above)
Comment (lpichler) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: 👍

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md (hunk 4 above)
Comment (lpichler) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: yes, repo

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md (hunk 6 above)
Comment (lpichler) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: oh yea, https://github.com/Ladas 👍

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md (hunk 8 above)
Comment (lpichler) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: No it is not 👍

**Comment by lpichler** [comment]
@JPrause @syncrou @imtayadeway updated. Thanks for review !

**Comment by syncrou** [comment]
👍 - LGTM

**Comment by JPrause** [comment]
@cybette we're ready for merge. Thanks!

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
+---
+title: "Last Week in ManageIQ: Yet Another move by Euwe"
+author: Libor Pichler
```
Comment (cybette) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: Please use github username as author, so when we implement linking to author it'll be easier!

File: site/_posts/2017-03-20-yet-another-move-by-euwe.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+tags: LWIMIQ
+---
+Hello World!
+
+My name is [Libor](https://github.com/lpichler "Can you guess what word is highlighted in my avatar?") and I am part of Platform Team in Brno.
+
+It's that time again. Time to recap of last week in ManageIQ!
+
+## Featured
+This week we were busy with moving last features to Euwe for upcoming 5.7.2 release!
```
Comment (cybette) [inline site/_posts/2017-03-20-yet-another-move-by-euwe.md]: 5.7.2 is downstream versioning, please use Euwe-3

**Comment by cybette** [comment]
@lpichler  Sorry for late review, just couple more things as mentioned!

**Comment by lpichler** [comment]
@cybette that's ok, updated 👍  thanks

**Pull request merged**